dbms/src/Debug/MockExecutor/astToExecutorUtils.cpp

```diff
@@ -42,7 +42,7 @@ ColumnName splitQualifiedName(const String & s)
 }
 
 
-std::__wrap_iter<const std::pair<std::string, TiDB::ColumnInfo> *> checkSchema(const DAGSchema & input, String checked_column)
+DAGSchema::const_iterator checkSchema(const DAGSchema & input, const String & checked_column)
 {
     auto ft = std::find_if(input.begin(), input.end(), [&](const auto & field) {
```

Comment:
```suggestion
    auto ft = std::find_if(input.begin(), input.end(), [&checked_column](const auto & field) {
```

emmm, and capture checked_column is enough?